engine/src/main/java/org/terasology/engine/world/chunks/blockdata/ExtraBlockDataManager.java

```diff
@@ -33,9 +33,9 @@
  * whether the field is applicable for that block. For example:
  *
  * <pre>
- * @ExtraDataSystem
+ * {@code @ExtraDataSystem}
```

Comment:
I think you can put the `{@code` right next to the `<pre>` and close it only in the end.

Reply:
nope, `{@code ...}` (same as other `@...` tags) unfortunately doesn't work across multiple lines.